### PR TITLE
Added logging to Snow AMI bootstrap scripts.

### PR DESCRIPTION
**Pull request by yxinchen**
*Description of changes:*
Added logging to Snow AMI bootstrap scripts.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap-after.sh [hunk 1]
```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
+source /etc/eks/logging.sh
 
-KUBE_VIP_IMAGE=$1
```
Comment (g-gaston) [inline projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap-after.sh:3]: We need to pass the kube-vip image as input so it's not hardcoded in the ami

File: projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap.sh [hunk 2]
```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
+source /etc/eks/logging.sh
+SCRIPT_LOG=/var/log/eks-bootstrap.log
+touch $SCRIPT_LOG
 
-KUBE_VIP_IMAGE=$1
```
Comment (g-gaston) [inline projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap.sh:3]: Same here for the kube-vip image 🙂

File: projects/kubernetes-sigs/image-builder/packer/ami/additional_files/logging.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (g-gaston) [inline projects/kubernetes-sigs/image-builder/packer/ami/additional_files/logging.sh:1]: This is a new file, in order to have it copied to the ami, can you add an entry [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/image-builder/packer/ami/ansible_extra_vars.yaml#L3)?

**Review by g-gaston — COMMENTED** [review]
Thanks a lot!

File: projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap-after.sh (hunk 1 above)
Comment (yxinchen) [reply projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap-after.sh:3]: added, thanks!

File: projects/kubernetes-sigs/image-builder/packer/ami/additional_files/logging.sh (hunk 3 above)
Comment (yxinchen) [reply projects/kubernetes-sigs/image-builder/packer/ami/additional_files/logging.sh:1]: added the entry, thanks.

**Review by g-gaston — APPROVED** [review]

**Pull request merged**